src/mozanalysis/experiment.py

```diff
@@ -1227,23 +1219,26 @@ def _validate_last_date_data_required(self, attribute, value):
 class AnalysisWindow:
     """Represents the range of days in which to measure a metric.
 
-    The range is measured in "days after enrollment", and is inclusive.
+    The range is measured in "days relative enrollment", and is inclusive.
 
-    For example, ``AnalysisWindow(0, 6)`` is the first week after enrollment.
+    For example, ``AnalysisWindow(0, 6)`` is the first week after enrollment
+    and `AnalysisWindow(-8,-1)` is the week before enrollument
 
     Args:
-        start (int): First day of the analysis window, in days since
-            enrollment.
+        start (int): First day of the analysis window, in days relative
+            enrollment. 0 indicates the date of enrollment. Positive numbers
+            are after enrollment, negative are before.
         end (int): Final day of the analysis window, in days since
```

Comment:
```suggestion
        end (int): Final day of the analysis window, in days relative to enrollment end.
```

Reply:
I'd also add something about how `end` must be the same sign as `start` (or 0). Unless that's not accurate, then don't say it :D

Reply:
Added!